### PR TITLE
2nd branch

### PR DESCRIPTION
**Pull request by larebsyed**

### CODE REVIEW

**Comment by larebsyed** [comment]
@rykabdullah7 Good job on completing this, few changes are requested
- The star should keep showing once clicked.

File: Project List/scripts.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const hide_buttons = document.getElementsByClassName("btn");
```
Comment (larebsyed) [inline Project List/scripts.js:1]: the name of variable should be in camelCase. `hide_button => hideButton`

File: Project List/scripts.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Array.from(hide_buttons).forEach((button) => {
+    button.addEventListener('click',() => {
+        const element = button.parentNode;
+        element.style.display = 'none';
+    } )
+})
+
+const unhid = document.getElementById("unhide");
```
Comment (larebsyed) [inline Project List/scripts.js:9]: use complete variable name `unhid => unhideButtons`

File: Project List/index.html
Comment (larebsyed) [inline Project List/index.html:1]: please rename the file to project-list

File: Project List/scripts.js (hunk 1 above)
Comment (rykabdullah7) [reply Project List/scripts.js:1]: Solved in the latest merge

File: Project List/scripts.js (hunk 2 above)
Comment (rykabdullah7) [reply Project List/scripts.js:9]: solved in the latest merge

File: Project List/index.html
Comment (rykabdullah7) [reply Project List/index.html:1]: solved in the latest merge

**Pull request merged**